Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -290,6 +291,11 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
             return properties;
         }
 
+        internal static string ResolveConnectionAreaMessage(string host) =>
+            host.EndsWith(".snowflakecomputing.cn", StringComparison.InvariantCultureIgnoreCase)

```

Comment:
should we check for com (current one) and all other domains too (as else branch)?

Reply:
agree @sfc-gh-dprzybysz, let's have it as here
https://github.com/snowflakedb/snowflake-jdbc/pull/1826

Reply:
Done